aws-android-sdk-core/build.gradle

```diff
@@ -20,14 +19,14 @@ android {
 }
 
 dependencies {
-    api 'com.google.code.gson:gson:2.2.4'
-    implementation 'com.fasterxml.jackson.core:jackson-core:2.1.1'
+    api 'com.google.code.gson:gson:2.8.6'
+    implementation 'com.fasterxml.jackson.core:jackson-core:2.11.0'
 
-    testImplementation 'joda-time:joda-time:2.0'
-    testImplementation 'junit:junit:4.12'
+    testImplementation 'joda-time:joda-time:2.8.1'
+    testImplementation 'junit:junit:4.13'
     testImplementation 'org.apache.commons:commons-io:1.3.2'
     testImplementation 'org.easymock:easymock:3.1'
-    testImplementation 'org.robolectric:robolectric:3.8'
+    testImplementation 'org.robolectric:robolectric:4.3.1'
```

Comment:
this may break some things. I distinctly remember having to set the version to 3.8 because of some incompatibilities in our code. Can you confirm that this is safe?

Reply:
We should also confirm with other robolectric dependencies throughout the repo

Reply:
The confirmation is that the unit tests are passing in the CI/CD! :-D

Reply:
It's possible that the issues were with the integration tests that are now removed. It sounds fine in that case.